src/tasks/importEvents.js

```diff
@@ -0,0 +1,102 @@
+const https = require('https');
+const request = require('request-promise-native');
+const { mobilizeAmericaToMongoDocument } = require('../transformers/event');
+const { setupDatabase } = require('../utils/connectToDatabase');
+
+function promotedOrganizationsUrl() {
+  return 'https://api.mobilize.us/v1/organizations/1316/promoted_organizations';
+}
+
+function organizationEventsUrl(organizationId) {
+  return `https://api.mobilize.us/v1/organizations/${organizationId}/events?timeslot_start=gte_now`;
+}
+
+const mobilizeAmericaApiKey = process.env.MOBILIZE_AMERICA_API_KEY;
+const upsertBatchSize = 10;
+
+async function mobilizeAmericaRequest(url) {
+  return request({
+    url,
+    json: true,
+    headers: {
+      Authorization: `Bearer ${mobilizeAmericaApiKey}`,
+    },
+  });
+}
+
+async function getPromotedOrganizations() {
+  const body = await mobilizeAmericaRequest(promotedOrganizationsUrl());
+  const organizationIds = body.data.map(({ id }) => id);
+  return organizationIds;
+}
+
+async function getEventsForOrganization(organizationId) {
+  let results = []
+  let nextUrl = organizationEventsUrl(organizationId);
+  while (nextUrl) {
+    console.log(nextUrl);
```

Comment:
Let's make sure we take all of the misc. `console.log`'s out or surround them with a logging flag, trying to parse through logs in AWS CloudWatch is not fun..

Reply:
oops yes